### PR TITLE
rebranding Smarp to Haiilo Advocacy

### PR DESCRIPTION
**Pull request by thanh-nguyen**
We recently had a rebranding, with new names, logos and websites. There's no update in the functionality.

---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [ ] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [ ] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ ] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [ ] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [ ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/Haiilo/apiDefinition.swagger.json [hunk 1]
```diff
@@ -4,8 +4,8 @@
   "definitions": {},
   "host": "smarp.smarpshare.com",
   "info": {
-    "description": "Smarp is a mobile-first employee communication platform to drive alignment, productivity and champion your brand. Connect to Smarp to manage your posts.",
-    "title": "Smarp",
+    "description": "Haiilo Advocacy is a mobile-first employee communication platform to drive alignment, productivity and champion your brand. Connect to Haiilo Advocacy to manage your posts.",
+    "title": "Haiilo Advocacy",
     "version": "1.0",
```
Comment (Amjed-Ayoub) [inline certified-connectors/Haiilo/apiDefinition.swagger.json:14]: @thanh-nguyen The "contact" property is required. Please see out template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiDefinition.swagger.json). 
```
    "contact": {
      "name": "Full name",
      "url": "Contact URL",
      "email": "Contact email address"
    }
```

File: certified-connectors/Haiilo/apiDefinition.swagger.json [hunk 2]
```diff
@@ -4,8 +4,8 @@
   "definitions": {},
```
Comment (Amjed-Ayoub) [inline certified-connectors/Haiilo/apiDefinition.swagger.json:4]: @thanh-nguyen The 'x-ms-connector-metadata' property is required. Please see out template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiDefinition.swagger.json).
```
  "x-ms-connector-metadata": [
    {
      "propertyName": "Website",
      "propertyValue": ""
    },
    {
      "propertyName": "Privacy policy",
      "propertyValue": ""
    },
    {
      "propertyName": "Categories",
      "propertyValue": ""
    }
  ]
```

File: certified-connectors/Haiilo/apiDefinition.swagger.json [hunk 3]
```diff
@@ -16,7 +16,7 @@
     "/publicapi/channel": {
       "get": {
         "description": "Retrieve channel list for creating a new post",
-        "operationId": "SmarpRetrieveChannelList",
+        "operationId": "AdvocacyRetrieveChannelList",
```
Comment (Amjed-Ayoub) [inline certified-connectors/Haiilo/apiDefinition.swagger.json:24]: @thanh-nguyen The 'default' response at line 33. should not have schema definition. Schemas should be defined on expected responses only.

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

**Comment by thanh-nguyen** [comment]
@microsoft-github-policy-service agree company="Haiilo Oy"

**Comment by thanh-nguyen** [comment]
@Amjed-Ayoub thanks for checking the PR. I've made changes according to the comments. Please check if they're sufficient.

**Comment by thanh-nguyen** [comment]
Hi @Amjed-Ayoub can you check if the latest changes are sufficient? Thanks

**Comment by vmanoharas** [comment]
Hello @thanh-nguyen 

The pipeline was experiencing internal issues, we have resolved them. Could you please resubmit your PR/commit so that it can be triggered again and undergo validation? Thank you for your understanding. We apologize any inconvenience caused.

**Pull request closed without merging**